src/Assignment-2/assignment2.css

```diff
@@ -8,60 +8,82 @@ height: 100%;
 body{
     border : 1px solid black;
 }
+ul li {
+    list-style: none;
+}
 header{
+    border-bottom: 1px solid black;
+}
+.page-header{
     border: 1px solid black;
     background:chartreuse;
     margin:10px;
+    height:230px;
 }
 
-img{
+.page-logo{
     width: 200px;
     height: 200px;
     padding-top:  20px;
     padding-bottom:10px;
 }
-form{
-    float:right;
-    margin:70px;
-    font-size: 10px;
-    
+
+.page-header form{
+    display: inline-block;
+    float: right;
+    margin-top: 70px;
+    margin-right: 20px;
+    font-size: 40px;
+}
+.page-header form input{
+    height:30px;
+
 }
+.links ul li{
+    display: inline;
+    margin-left:80px;
 
+}
 .links{
     border:1px solid black;
-    font-size: 40px;
+    background-color: coral;
     margin:10px;
-    padding:5px;
-    text-align:center;
-
+    text-align: center;
 }
-.mainbody{
```

Comment:
To give two same styling to more than 1 class, we do it like below:
.class1name, .class2name{

}